diagnostics/ENSO_RWS/shared/set_variables_AM4.py

```diff
@@ -0,0 +1,22 @@
+import os
```

Comment:
@jhafner2 You could combine this with set_variables_CESM.py and set_variables_CMIP.py by creating a nested dictionary for the environment variable names and their definitions, then looping through the keys and values (e.g.,):

```
def recursive_dict_items(dictionary, case=""):
    for key, value in dictionary.items():
        if type(value) is dict:
            if (key==case):
                yield from recursive_dict_items(value)

var_dict = {
      'am4':{'ta_var': 'ta',
            'prw_var': 'prw'
      },
      'cesm':{'ua_var': 'U',
              'va_var': 'V'
      },
      'cmip': {'ts_var': 'ts',
               'pr_var': 'pr'
      }
}

case =['am4','cesm','cmip']
for c in case :
    for key, value in recursive_dict_items(var_dict, c):
       #print(key, value)
       os.environ[key] = value
```